que.go

```diff
@@ -155,38 +156,53 @@ func execEnqueue(j *Job, q queryable) error {
 		return ErrMissingType
 	}
 
-	queue := pgx.NullString{
+	var status pgtype.Status
+	if j.Queue != "" {
+		status = pgtype.Present
+	} else {
+		status = pgtype.Null
+	}
+
+	queue := &pgtype.Text{
```

Comment:
@cbandy its required that pgtype values be pointers, please refer [here](https://github.com/jackc/pgx/issues/292).